HW1-Code.py

```diff
@@ -20,79 +27,91 @@
 # max      7.900000    4.400000    6.900000    2.500000
 
 
-#Plots:
-#1)Scatter plot
+##################################### Plots:
+##  1)Scatter plot
+# 		add column headings to have something to anchor on when graphing
 iris_df.columns = ['Sepal Length','Sepal Width', 'Petal Length', 'Petal Width', 'Class']
 iris_df.head
-
-
+#		plot here
 fig = px.scatter(iris_df, x="Sepal Length", y="Sepal Width", color="Class", title= "Iris Test Plot")
 fig.show()
 
+##  2) Histogram
+#		group by class and have a different color for each class
 fig0 = px.histogram(iris_df, x="Class", color="Class")
 fig0.show()
 
-
-
-#works but messy
+##  3) Line graph
+#		group by class and have a different color for each class
+#       This looks messy because it was a line graph that had to go through each point in the scatterplot. I will have to refine this later to just reflect a trendline.

```

Comment:
Awesome, this makes it easier to understand. I would just hit enter after the period (...scatterplot.) so that its not cut out from the viewer.